src/term.rs

```diff
@@ -397,7 +397,14 @@ impl Term {
             }
             Term::Fun(_, _) => String::from("<func>"),
             Term::Lbl(_) => String::from("<label>"),
-            Term::Enum(s) => format!("`{}", s),
+            Term::Enum(Ident(s, _)) => {
```

Comment:
We should avoid matching on the actual structure of `Ident`, because it may very well change in the future. Instead, rather match an `id` and then use conversion methods like `to_string`.

Reply:
Ok fixing it in next commit.